### DISCUSSION TITLE
Move from ISO 639-1 to IETF language tag

### DISCUSSION THREAD

**Discussion by MAGICCC** [WeblateOrg/weblate#7996] [Q&A] [answered]
Hello,

We may change with our project from ISO 639-1 to IETF language tags (see example table below). Is that possible to have a smooth transition inside weblate? I searched a bit but couldnt really find an info on that.
When I go inside Weblate and trying to add a new language, e.g. da-dk, it wont anything.

| **Old Code** | **New Code** | **Display Name**               |
| ------------ | ------------ | ------------------------------ |
| cs           | cs-cz        | Čeština (Czech)                |
| da           | da-dk        | Danish (Dansk)                 |
| de           | de-de        | Deutsch (German)               |
| en           | en-gb        | English                        |
| es           | es-es        | Español (Spanish)              |
| fi           | fi-fi        | Suomi (Finish)                 |
| fr           | fr-fr        | Français (French)              |
| hu           | hu-hu        | Magyar (Hungarian)             |
| it           | it-it        | Italiano (Italian)             |
| ko           | ko-kr        | 한국어 (Korean)                |
| lv           | lv-lv        | latviešu (Latvian)             |
| nl           | nl-nl        | Nederlands (Dutch)             |
| pl           | pl-pl        | Język Polski (Polish)          |
| pt           | pt-pt        | Português (Portuguese)         |
| ro           | ro-ro        | Română (Romanian)              |
| ru           | ru-ru        | Pусский (Russian)              |
| sk           | sk-sk        | Slovenčina (Slovak)            |
| sv           | sv-se        | Svenska (Swedish)              |
| uk           | uk-ua        | Українська (Ukrainian)         |
| zh           | zh-cn        | 简体中文 (Simplified Chinese)  |
|              | zh-tw        | 繁體中文 (Traditional Chinese) |

**Comment by nijel** [comment, marked as answer]
If you rename the files it will work just fine in Weblate , see https://docs.weblate.org/en/latest/admin/languages.html#parsing-language-codes for more details

**Reply by MAGICCC** [reply]
Thanks! I'll will try to edit the Language code in /language/

**Reply by nijel** [reply]
There is no need to rename languages in Weblate, it will match changed files to correct codes.
